recipes/snp-pipeline/meta.yaml

```diff
@@ -14,7 +14,6 @@ requirements:
   build:
     - python
     - pyvcf >=0.6.7
-    - setuptools
```

Comment:
This shouldn't be removed.

Reply:
@mbargull Thank you for the review.  May I ask, what are the guidelines for deciding which packages are build requirements?

Reply:
You're welcome.
The build requirements can be very individual for each package. But for simple Python packages, like the one at hand, it is quite easy to determine:
- need `python`
- need anything that's not part of Python's stdlib which is imported in `setup.py`
- need anything specified in `setup(...)`'s `setup_requires` parameter

Exceptions may occur, like for example:
 - if `setup.py` imports the package itself, most likely it's best to add all run requirements as build requirements, too
 - if some extensions are built, then additional compiler packages are required

In `snp-pipline`'s case we thus only need `python` and `setuptools`, i.e. `pyvcf`, `psutil` and `biopython` can be removed from build requirements.